job-runner/chain/traverser.go

```diff
@@ -377,3 +386,36 @@ func (t *traverser) sendJL(pJob proto.Job, err error) {
 		jLogger.Errorf("problem sending job log (%#v) to the RM: %s", jl, err)
 	}
 }
+
+func (t *traverser) retrySequence(j proto.Job) {
+	sequenceStartJob := t.chain.SequenceStartJob(j)
+
+	// Create list of jobs in sequence that have been ran and we need to undo
```

Comment:
What does it mean to "undo" a job? I have a pretty good idea, but what questions/possibilities might arise in the mind of future engineers? For example, "undo" could mean "rollback" somehow (as if jobs could undo their work in preparation for re-doing it).

Reply:
undo means to make the system think it has to run the job again by:

* making job pending again
* decrement those # of jobs from count in chain

I can rename this to `jobsToRetry`